diff --git a/Trio/Sources/Modules/Home/View/Chart/ChartElements/ForecastView.swift b/Trio/Sources/Modules/Home/View/Chart/ChartElements/ForecastView.swift
index 0cf9d5014..6413f3342 100644
--- a/Trio/Sources/Modules/Home/View/Chart/ChartElements/ForecastView.swift
+++ b/Trio/Sources/Modules/Home/View/Chart/ChartElements/ForecastView.swift
@@ -41,7 +41,7 @@ struct ForecastView: ChartContent {
                         Decimal(minForecast[index] + 1)
                         .asMmolL
 
-                    if xValue <= Date(timeIntervalSinceNow: TimeInterval(hours: 2.5)) {
+                    if xValue >= Date(), xValue <= Date(timeIntervalSinceNow: TimeInterval(hours: 2.5)) {
                         AreaMark(
                             x: .value("Time", xValue),
                             // maxValue is already parsed to user units, no need to parse
@@ -57,7 +57,7 @@ struct ForecastView: ChartContent {
                     let yMaxValue = units == .mgdL ? Decimal(maxForecast[index]) : Decimal(maxForecast[index])
                         .asMmolL
 
-                    if xValue <= Date(timeIntervalSinceNow: TimeInterval(hours: 2.5)) {
+                    if xValue >= Date(), xValue <= Date(timeIntervalSinceNow: TimeInterval(hours: 2.5)) {
                         AreaMark(
                             x: .value("Time", xValue),
                             // maxValue is already parsed to user units, no need to parse
@@ -80,7 +80,7 @@ struct ForecastView: ChartContent {
             let displayValue = units == .mmolL ? valueAsDecimal.asMmolL : valueAsDecimal
             let xValue = timeForIndex(forecastValue.index)
 
-            if xValue <= Date(timeIntervalSinceNow: TimeInterval(hours: 2.5)) {
+            if xValue >= Date(), xValue <= Date(timeIntervalSinceNow: TimeInterval(hours: 2.5)) {
                 LineMark(
                     x: .value("Time", xValue),
                     y: .value("Value", displayValue)
